test/transmit.js

```diff
@@ -1194,7 +1196,8 @@ describe('transmission', () => {
             await log;
         });
 
-        it('stops processing the stream when the connection closes', async () => {
+        // NOTE do not merge with this skip. Can be removed once fix in node v17.1.0 is released.
+        it.skip('stops processing the stream when the connection closes', async () => {
```

Comment:
Node v17.1.0 came out today with the fix for https://github.com/nodejs/node/issues/40528, so this skip shouldn't be needed anymore.  Fingers crossed GH actions pulls the latest version of v17 in the next test run 😆 
```suggestion
        it('stops processing the stream when the connection closes', async () => {
```